modules/rules_license/0.0.2/source.json

```diff
@@ -0,0 +1,4 @@
+{
+    "integrity": "sha256-pe3/yBDHTjKpp+9flZG8BdE2KyRLniIyPzjLu6ukEoE=",
+    "url": "https://mirror.bazel.build/github.com/bazelbuild/rules_license/releases/download/0.0.2/rules_license-0.0.2.tar.gz"
```

Comment:
this URL should be the original source archive from github. the mirror portion will be automatically prepended.